doc/src/cli/config.md

```diff
@@ -100,6 +100,18 @@ and use the `--commit` flag to commit changes when needed.
 - `true` (default)
 - `false`
 
+### spice.branchCreate.prefix
+
+<!-- gs:version v0.14.0 -->
```

Comment:
used `0.14.0` since that looks like the next version that will get cut.

Reply:
looks like this actually causes a build error. Is there an alias to use for unreleased versions?